### PR TITLE
add explicit per-thread mutexset and stack

### PR DESCRIPTION
**Pull request by dvyukov**

### CODE REVIEW

File: mm/ktsan/ktsan.c [hunk 1]
```diff
@@ -325,10 +360,6 @@ EXPORT_SYMBOL(ktsan_report_enable);
 void ktsan_sync_acquire(void *addr)
 {
 	ENTER(KT_ENTER_NORMAL);
-#if KT_DEBUG
-	kt_trace_add_event(thr, kt_event_type_acquire, kt_pc_compress(pc));
-	kt_clk_tick(&thr->clk, thr->id);
-#endif /* KT_DEBUG */
```
Comment (xairy) [inline mm/ktsan/ktsan.c]: Why did you remove this?

Comment (dvyukov) [inline mm/ktsan/ktsan.c]: This is moved inside of kt_sync_acquire

**Comment by xairy** [comment]
LGTM
